sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -7,10 +7,25 @@
  */
 
 /**
- * @defgroup    net_gnrc_gomach A traffic-adaptive multi-channel MAC
+ * @defgroup    net_gnrc_gomach GoMacH
  * @ingroup     net_gnrc
- * @brief       A traffic adaptive MAC protocol that provides high traffic
- *              adaptability, high energy efficiency and high robustness.
+ * @brief       GoMacH is, "a General, nearly Optimal MAC protocol
```

Comment:
sorry, but this is not _brief_, maybe the original defgroup description (`A traffic-adaptive multi-channel MAC`) can be (re) used here. Please note that the `@brief` sentence will be displayed in doxygen below the group name (GoMacH, here), hence it should be a one-liner. You can use this longer text as as the first paragraph of the detailed description (above the feature list), though.

Reply:
to make my point a bit clearer, look at the module [page](https://doc.riot-os.org/modules.html) in doxygen, the `@brief` for the boards group is for instance `Board specific definitions and implementations`.

Reply:
@zhuoshuguo you can also look at the compiled doc locally by building with `make doc` in the RIOT root directory. The results will be in `doc/doxygen/html/`. Choose the `module.html` to look at the new version of the page @smlng mentioned.

Reply:
(saw this comment only after ACKing)